flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/utils/PlannerMocks.java

```diff
@@ -69,8 +77,25 @@ public static FlinkPlannerImpl createDefaultPlanner() {
                         functionCatalog.asLookup(parser::parseIdentifier),
                         catalogManager.getDataTypeFactory(),
                         parser::parseSqlExpression));
+    }
+
+    public FlinkPlannerImpl getPlanner() {
         return planner;
     }
 
-    private PlannerMocks() {}
+    public ParserImpl getParser() {
+        return parser;
+    }
+
+    public CatalogManager getCatalogManager() {
+        return catalogManager;
+    }
+
+    public Catalog getCurrentCatalog() {
+        return catalogManager.getCatalog(catalogManager.getCurrentCatalog()).get();
+    }
+
+    public static PlannerMocks createDefault() {
```

Comment:
nit: there is only the default, and nothing else. We could also just name this `create`.